examples/pytorch/vision/Face_Detection/data/config_qvga.py

```diff
@@ -54,11 +54,12 @@
 _C.USE_NMS = True
 
 # dataset config
-_C.HOME = '/mnt/'
+_C.HOME = '/mnt'
 
 # face config
 _C.FACE = EasyDict()
 _C.FACE.TRAIN_FILE = './data/face_train.txt'
 _C.FACE.VAL_FILE = './data/face_val.txt'
 _C.FACE.WIDER_DIR = '/mnt/WIDER_FACE'
```

Comment:
I think it's better to make this `_C.FACE.WIDER_DIR =  _C.HOME + '/WIDER_FACE'` here and in `config.py`. Also, I don't think this is needed to be set by the user when the instructions mention that the folder must be named `WIDER_FACE`.